### ISSUE TITLE
0011902:
Mail account can't send to web.de anymore (SMTP, StartTLS, plain)

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#5889]
**Reported by lab-at-nohl on 7 Jun 2016 10:35**

**Version:** Egon Community Edition (2016.03.2)

For additional accoutns web.de freemail stopped working suddenly. UI is saying &#039;Could not read from smtp.web.de&#039;. It does not affect sending from system accounts to web.de - postfix still delivers to web.de successfully. 

For me it looks like the server isn&#039;t responding anything and the connection times out (Zend/Mail/Protocol/Abstract.php, line 400). I don&#039;t know if it is related to Tine 2.0 for sure because it worked a week ago; maybe the server blocks my subnet. But I still want to make sure that Tine 2.0 is compliant to web.de. Can anybody confirm this?

**Additional information:** 0fb7d 8996c test - 2016-06-03T13:47:33+00:00 INFO (6): Tinebase_Smtp::createDefaultTransport::74 Setting default SMTP transport. Hostname: localhost
0fb7d 8996c test - 2016-06-03T13:47:33+00:00 INFO (6): Tinebase_Smtp::setDefaultTransport::111 Setting SMTP transport: Zend_Mail_Transport_Smtp
0fb7d 8996c test - 2016-06-03T13:49:33+00:00 WARN (4): Felamimail_Controller_Message_Send::sendMessage::81 Could not send message: exception &#039;Zend_Mail_Protocol_Exception&#039; with message &#039;Could not read from smtp.web.de&#039; in /srv/tine/Zend/Mail/Protocol/Abstract.php:400
Stack trace:
#0 /srv/tine/Zend/Mail/Protocol/Abstract.php(430): Zend_Mail_Protocol_Abstract-&gt;_receive(NULL)
#1 /srv/tine/vendor/zendframework/zendframework1/library/Zend/Mail/Protocol/Smtp/Auth/Plain.php(93): Zend_Mail_Protocol_Abstract-&gt;_expect(235)
#2 /srv/tine/vendor/zendframework/zendframework1/library/Zend/Mail/Protocol/Smtp.php(217): Zend_Mail_Protocol_Smtp_Auth_Plain-&gt;auth()
#3 /srv/tine/vendor/zendframework/zendframework1/library/Zend/Mail/Transport/Smtp.php(200): Zend_Mail_Protocol_Smtp-&gt;helo(&#039;localhost&#039;)
#4 /srv/tine/vendor/zendframework/zendframework1/library/Zend/Mail/Transport/Abstract.php(348): Zend_Mail_Transport_Smtp-&gt;_sendMail()
#5 /srv/tine/vendor/zendframework/zendframework1/library/Zend/Mail.php(1194): Zend_Mail_Transport_Abstract-&gt;send(Object(Tinebase_Mail))
#6 /srv/tine/Tinebase/Smtp.php(145): Zend_Mail-&gt;send(Object(Felamimail_Transport))
#7 /srv/tine/Felamimail/Controller/Message/Send.php(242): Tinebase_Smtp-&gt;sendMessage(Object(Tinebase_Mail), Object(Felamimail_Transport))
#8 /srv/tine/Felamimail/Controller/Message/Send.php(79): Felamimail_Controller_Message_Send-&gt;_sendMailViaTransport(Object(Tinebase_Mail), Object(Felamimail_Model_Account), Object(Felamimail_Model_Message), true, Array)
#9 /srv/tine/Felamimail/Frontend/Json.php(241): Felamimail_Controller_Message_Send-&gt;sendMessage(Object(Felamimail_Model_Message))
#10 [internal function]: Felamimail_Frontend_Json-&gt;saveMessage(Array)
#11 /srv/tine/vendor/zendframework/zendframework1/library/Zend/Server/Abstract.php(232): call_user_func_array(Array, Array)
#12 /srv/tine/Zend/Json/Server.php(589): Zend_Server_Abstract-&gt;_dispatch(Object(Zend_Server_Method_Definition), Array)
#13 /srv/tine/Zend/Json/Server.php(197): Zend_Json_Server-&gt;_handle()
#14 /srv/tine/Tinebase/Server/Json.php(293): Zend_Json_Server-&gt;handle(Object(Zend_Json_Server_Request))
#15 /srv/tine/Tinebase/Server/Json.php(166): Tinebase_Server_Json-&gt;_handle(Object(Zend_Json_Server_Request))
#16 /srv/tine/Tinebase/Core.php(229): Tinebase_Server_Json-&gt;handle(Object(Zend\Http\PhpEnvironment\Request))
#17 /srv/tine/index.php(18): Tinebase_Core::dispatchRequest()
#18 {main}
0fb7d 8996c test - 2016-06-03T13:49:33+00:00 DEBUG (7): Tinebase_Translation::getTranslation::305 locale used: Felamimail/de
0fb7d 8996c test - 2016-06-03T13:49:33+00:00 DEBUG (7): Tinebase_Translation::getTranslation::305 locale used: Tinebase/de
0fb7d 8996c test - 2016-06-03T13:49:33+00:00 WARN (4): Tinebase_Server_Json::_handleException::325 Tinebase_Exception_SystemGeneric -&gt; Fehler: Could not read from smtp.web.de
0fb7d 8996c test - 2016-06-03T13:49:33+00:00 ERR (3): Tinebase_Exception::log::104 Tinebase_Exception_SystemGeneric -&gt; Fehler: Could not read from smtp.web.de
0fb7d 8996c test - 2016-06-03T13:49:33+00:00 INFO (6): index.php (25) METHOD: Tinebase_Server_Json::Felamimail.saveMessage / TIME: 2m /  Memory usage: 7.5 MB /  Real patch cache size: 16275 / PID: 23415

**Comment by Gloirin** [comment]
**Comment posted by weberho on 7 Jun 2016 11:05**

This might be related to a changed SPF setting of the provider (read more at http://www.heise.de/newsticker/meldung/United-Internet-verschaerft-Spam-Bekaempfung-3225281.html ). Do you use the submission port 587 - if not try to use it.

**Comment by Gloirin** [comment]
**Comment posted by lab-at-nohl on 8 Jun 2016 13:49**

Yes I did use port 587 (authenticated users aren&#039;t accepted at port 25 anyway, I guess). I wrote to the postmaster of web.de, they didn&#039;t reply. Thanks to your link I know the background now. 

But it seems to work again. Propably misconfiguration of *submissions* in consequence of the invention of SPF for *transport/transport* wich IMHO isn&#039;t a good idea anyway. 

I&#039;ll close this. Thank you.
